docs/schedule-key-swap.md

```diff
@@ -0,0 +1,122 @@
+
+If you decide to use this feature, then run the commands described in this guide **before** the Bittensor mainnet opens.
+
+:::tip Window to use this schedule coldkey swap feature
+
+You can use this schedule cold key swap feature **only in the window** starting Monday July 8, 5 PM US Eastern and ending when the mainnet opens fully on Thursday July 11, 5 PM US Eastern.
+:::
+
+## Description
+
+The schedule coldkey swap feature works like this:
+
+- The schedule coldkey swap feature enables you to only schedule the actual swapping of your TAO funds from your old potentially compromised coldkey to a new coldkey. 
+- When you use this feature, it will not actually swap your TAO funds. It will only schedule the swap event. 
+- All scheduled coldkey swaps will be executed on-chain when the mainnet opens for normal operations on Thursday July 11, 5 PM US Eastern.
+
+## Do not use this feature if
+
+You may not need to use this feature. Do not use this feature if any of the following are true for you:
+
+- If you never used `btcli`, regardless of the Bittensor version, then do not use this feature. Your funds are safe. 
+- If you never used Bittensor version 6.12.2, then do not use this feature. Your funds are safe. 
+- If you only used any wallet browser extension or only a wallet application such as below, then your funds are safe:
+  - The Bittensor Wallet (for Chrome or iOS)
+  - SafeWallet, Talisman, Polkadot Vault, Tensor Wallet, Nova Wallet, polkadot.js
+  - Websites that utilize polkadot.js such as Taostats.io for staking unstaking
+- If you hold your TAO only on exchanges, then your funds are safe. 
+- If you installed Bittensor 6.12.2 not via PyPi package but directly from GitHub source, then do not use this feature. **Only the PyPi package of Bittensor 6.12.2 is malicious.**
+
+### Use this feature only if 
+
+Use this feature ONLY: 
+
+- If you downloaded the Bittensor PyPi package for the Bittensor version 6.12.2 and then performed any of the below operations.
+
+  **Impacted btcli 6.12.2 operations**
+
+  ```bash
+  btcli stake add
+  btcli stake remove
+  btcli wallet transfer
+  btcli root delegate
+  btcli root undelegate
+  btcli root set_take
+  btcli subnet register
+  ```
+
+:::important reminder
+If you installed Bittensor 6.12.2 not via PyPi package but directly from GitHub source, then do not use this feature. Only the PyPi package of Bittensor 6.12.2 is malicious.
+:::
+
+## Installing the schedule coldkey swap feature
+
+The schedule coldkey swap feature is available only at the below specified subtensor and bittensor repo branches:
+
+- **Subtensor repo**: Make sure to checkout the tag `specify release tag` on the Subtensor repo `https://github.com/opentensor/subtensor`.
+- **Bittensor repo and branch**: 
+    1. First make sure you do `git pull master` to pull the latest changes from the Bittensor repo `https://github.com/opentensor/bittensor`. 
```

Comment:
Question: is this step necessary, or can a person just use the bin/bash command from their terminal?  I'm asking as a not-very-familiar-with-GitHub person.

Reply:
Best to be safe and pull the latest.

Reply:
As a non GitHub person, I tried "git pull master" and got an error (there is no .git subdirectory in my .bittensor subdirectory), so it did not work, and predictably, the bin/bash command by itself installed 7.2.0.

Reply:
Okay, nm, I figured out my issue.  PEBKAC

Reply:
Okay, update.  On my Mac, `git pull master` did not work.  I needed to use `git pull origin master` (the "origin" was required), after double-checking with `git remove -v`.  It might just be a Mac thing.